{{cookiecutter.repostory_name}}/nginx/templates/default.conf.template

```diff
@@ -160,6 +164,14 @@ server {
         proxy_pass http://app:8000/metrics;
     }
 
+    location /business-metrics {
```

Comment:
I guess it's not a real issue here, but `location` without a trailing slash can lead to path traversing ( random google result: https://rioasmara.com/2021/12/27/path-traversal-via-nginx-alias/ )